backend/poetry.lock

```diff
@@ -2667,7 +2667,7 @@ version = "2.8.4"
 [[package]]
 category = "main"
 description = "Run a subprocess in a pseudo terminal"
-marker = "python_version >= \"3.4\" and sys_platform != \"win32\" or sys_platform != \"win32\""
+marker = "python_version >= \"3.4\" and sys_platform != \"win32\" or sys_platform != \"win32\" or python_version >= \"3.4\" and sys_platform != \"win32\" and (python_version >= \"3.4\" and sys_platform != \"win32\" or sys_platform != \"win32\")"
```

Comment:
hm, it's odd it is adding this for you. It doesn't add it when I run ```poetry update```. What version of poetry are you using?

Reply:
I use poetry 1.0.5 and ```poetry update query-parser``` was the only command I run.

Reply:
Can you try with 1.0.0?